### PR TITLE
Added a strain relief that mounts a 10 pin micro fit connector on the hotend.

### PR DESCRIPTION
**Pull request by lococnc**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

### CODE REVIEW

File: printer_mods/LoCoCNC/Strain_Relief_w_Microfit/Images/readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Images
```
Comment (KiloQubit) [inline printer_mods/LoCoCNC/Strain_Relief_w_Microfit/Images/readme.md:1]: Please delete this almost-empty file.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

The part is not printable without supports.  All Voron parts and mods must be printable without supports, so please redesign this to make it more printable.  The bottoms of these print in mid-air:

Screen Shot 2021-06-01 at 19 55 52

This overhang is steeper than 45 degrees, please ensure overhangs are no steeper than 45 degrees:

Screen Shot 2021-06-01 at 19 54 09

**Comment by lococnc** [comment]
What about putting a support in that the user is meant to cut away with
flush cutters? Kind of like the cable route on the 0.0?

**Comment by KiloQubit** [comment]
> What about putting a support in that the user is meant to cut away with flush cutters? Kind of like the cable route on the 0.0?
> […](#)

A built-in support is possible if there's no other option.  So you can either modify the mod to not require supports (maybe it can be two parts or have a different geometry), or you can have a built-in support.  If you have a built-in support, make sure that the README tells the user to remove the supports after printing.  The preference is to not require supports at all though, unless absolutely necessary - just like stock Voron parts.

File: printer_mods/LoCoCNC/Strain_Relief_w_Microfit/Images/readme.md (hunk 1 above)
Comment (lococnc) [reply printer_mods/LoCoCNC/Strain_Relief_w_Microfit/Images/readme.md]: Done

**Comment by lococnc** [comment]
Updated to not require supports. Added pic to show area to cut-away with flush cutters after prinitng.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the update!  Unfortunately some parts of this STL are still not printable as is (circled):

Screen Shot 2021-06-04 at 20 53 49

**Comment by lococnc** [comment]
Updated and test printed with no supports on V0.1

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Pull request merged**